src/main/scala/com/gu/invoicing/refundErroneousPayment/Model.scala

```diff
@@ -0,0 +1,84 @@
+package com.gu.invoicing.refundErroneousPayment
+
+import com.gu.invoicing.common.JsonSupport
+
+import java.time.LocalDate
+
+/**
+  * Data models and JSON codecs
+  */
+object Model extends JsonSupport {
+
+  case class Config(clientId: String, clientSecret: String)
+
+  case class AccessToken(access_token: String)
+
+  case class Invoice(Id: String,
+                     InvoiceNumber: String,
+                     Amount: BigDecimal,
+                     Balance: BigDecimal,
+                     PaymentAmount: BigDecimal,
+                     TargetDate: LocalDate,
+                     InvoiceDate: LocalDate,
+                     Status: String)
+
+  case class InvoiceQueryResult(records: List[Invoice])
+
+  case class Metrics(balance: BigDecimal, totalInvoiceBalance: BigDecimal, creditBalance: BigDecimal)
+  case class Account(metrics: Metrics)
+
+  case class RefundResult(Id: String)
+
+  case class Refund(
+      RefundNumber: String,
+      GatewayState: String,
+      RefundDate: LocalDate,
+      ReasonCode: String,
+      GatewayResponse: String,
+      Amount: BigDecimal,
+      Comment: String,
+      Status: String,
+      Gateway: String,
+      MethodType: String,
+      GatewayResponseCode: String,
+      Id: String
+  )
```

Comment:
minor - the indentation is inconsistent, most places seem to be 2 space, but here has 4 space and about 20 space in the same file.

Reply:
Thanks for pointing it out.  I've now added a scalafmt config file to the project and reformatted this file.